imdown-mobile/squads/add_squad.tsx

```diff
@@ -15,12 +17,36 @@ const AddSquadModal = (props: OwnProps) => {
     const [squadName, setSquadName] = useState("")
     const [showSquadEmoji, setShowSquadEmoji] = useState(false)
     const [showEmojiPicker, setShowEmojiPicker] = useState(false);
-    const [emojiPicked, setEmojiPicked] = useState(DEFAULT_EMOJI ); 
+    const [emojiPicked, setEmojiPicked] = useState(DEFAULT_EMOJI);
 
+    const addSquadOnBackend = () => {
```

Comment:
Don't make raw requests to the backend using `fetch`, utilize the `callBackend` function in `backend/backend.tsx` so that we can properly send session cookies

Reply:
This shouldn't even work except that there's a bug where the session cookies get cached once you send them so every time you make a request after that they get sent

Reply:
done, didnt realize you'd merged this code in already